torch/autograd/__init__.py

```diff
@@ -152,5 +152,5 @@ def grad(outputs, inputs, grad_outputs=None, retain_graph=None, create_graph=Non
         outputs, grad_outputs, retain_graph,
         inputs, only_inputs)
 
-status = torch._C._autograd_init()
-assert status, "Autograd failed to initialize."
+if not torch._C._autograd_init():
```

Comment:
It is, but this statement is executed only once, so and the overhead of this single boolean comparison is completely irrelevant